lib/src/lib.rs

```diff
@@ -139,6 +139,9 @@ pub mod syn;
 
 #[doc(inline)]
 pub use api::engine;
+#[cfg(feature = "protocol-http")]
+#[doc(inline)]
+pub use api::headers;
```

Comment:
Do we expect users to use these headers or we are only exporting them for the server?

Reply:
As we discussed, I have kept the header constants out of the public API in [3aa88f7](https://github.com/surrealdb/surrealdb/pull/2985/commits/3aa88f7cfd52825d06d4789f0fd919bfe740f359).